src/datanode/src/datanode.rs

```diff
@@ -223,6 +223,8 @@ pub struct FlushConfig {
     /// Interval to auto flush a region if it has not flushed yet.
     #[serde(with = "humantime_serde")]
     pub auto_flush_interval: Duration,
+    /// Global write buffer size for all regions.
+    pub global_write_buffer_size: Option<ReadableSize>,
```

Comment:
A bit off-topic: maybe we should distinguish between `xxxConfig` and `xxxOption`, for that `Option` is a provided value, it may contain optional fields, while `Config` is the resolved value, it won't have "maybe present" value, all of its fields should be either specified or fallback to some default value when implementing `From<Option>` for `Config`